regulations/tasks.py

```diff
@@ -38,15 +33,42 @@ def submit_comment(comment):
 
 
 @contextlib.contextmanager
-def TemporaryDirectory(*args, **kwargs):
+def AttachmentSet(comment):
```

Comment:
I wasn't aware of the reason `TemporaryDirectory` was CamelCase. I stuck to the pattern because `AttachmentSet` behaves like a constructor. I'll fix.